### ISSUE TITLE
How to populate the local databse?

### ISSUE THREAD

**Issue by JustAdreamerFL** [matejtarca/5proti5#2]
Hi, I am struggling to populate the databse. I am using a GUI app called "DB Browser for SQLite", but anytime I want to put some new record into a table, it gives me this type of error:

![Screenshot 2024-12-28 at 17 41 43](https://github.com/user-attachments/assets/7ac2db56-dc39-4927-97a6-4ff4075ef76b)

I wish to know, how did you populate your database?

**Comment by matejtarca** [comment]
Hey, it looks like you don't have the Game with id 1 created in the Game table so the foreign key check on gameId 1 fails.

I recommend first inserting into the Game table and only then creating teams for the given game.

Bear in mind I made up the schema very quickly for my usecase so if the DB schema is not to your liking feel free to submit PR with changes and I will review it.

**Comment by JustAdreamerFL** [comment]
Hey thanks for the quick reply! And also thank you for the instructions. Now I can populate the database. Maybe I could add this to readme.md for future users?

**Comment by matejtarca** [comment]
Yep, I think you are the first one to reuse this project so if you have any ideas for improving the readme fill free to contribute!
